documentation/api/development.md

```diff
@@ -19,10 +19,23 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
    curl -sSL https://install.python-poetry.org | python3 -
    ```
 
+   - MD- I ended up accomplishing this via
```

Comment:
@chouinar I got this to work, but maybe not via the best route. Calling ```python3``` didn't seem to honor my env settings?

Reply:
Hmm, not sure why any of these docs are specifically saying `python3` for the commands. That hasn't been needed in years. I believe pyenv just swaps out the `python` command. I think some of these docs could really use another pass, some of this is going to lead to people setting their env up in easy-to-break ways.

I'd strongly recommend only ever letting pyenv manage your python installations as I've found things tend to break if you do it yourself - and in ways that are incredibly difficult to fix.

Reply:
Looking at this a bit more, it seems pyenv does update both the `python` and `python3` command.

Can you explain a bit more about what wasn't getting picked up, or what errors you were seeing?